website/source/docs/auth/radius.html.md

```diff
@@ -0,0 +1,309 @@
+---
+layout: "docs"
+page_title: "Auth Backend: RADIUS"
+sidebar_current: "docs-auth-radius"
+description: |-
+  The "radius" auth backend allows users to authenticate with Vault using an existing RADIUS server.
+---
+
+# Auth Backend: RADIUS
+
+Name: `radius`
+
+The "radius" auth backend allows users to authenticate with Vault using
+an existing RADIUS server that accepts the PAP authentication scheme. 
+
+The mapping of users to Vault policies is managed by using the
+`users/` path.
+
+Optionally, a configurable set of policies can be granted to all users that 
+can successfully authenticate but are not registered in the `users/` path.
+
+## Authentication
+
+The 'radius' auth backend accepts 
```

Comment:
Can we rephrase the line since it seems to abruptly stop. Or we could just remove it.